src/test/java/api/support/fixtures/policies/PoliciesActivation.java

```diff
@@ -0,0 +1,18 @@
+package api.support.fixtures.policies;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@AllArgsConstructor
+@Getter
+public final class PoliciesActivation {
```

Comment:
What about this structure is specific to activation?

Could this be called `Policies` instead?

If it is specific to activation, is it the policies that are due to be activated, the ones that have been activated, or something else?

Reply:
This is policies that are going to be activated. Maybe `PoliciesToActivate`?

Reply:
Sure, I'm good with `PoliciesToActivate`